.lgtm.yml

```diff
@@ -0,0 +1,7 @@
+# https://lgtm.com/projects/g/freeipa/freeipa
+
+queries:
+  # False positive unmatchable dollar in regular expression, e.g.
+  # installutils.update_file(certmap_filename, '$ISSUER_DN', str(ca_subject))
+  - exclude: py/regex/unmatchable-dollar
```

Comment:
Is there a way we can disable this check in just the one place rather than project-wide?

Also, it is a bug in *LGTM* (doesn't recognise that the datum flows through `re.escape`).
Will you report it to your acquaintant or suggest the best way to report the issue?

Reply:
Yes, I figured it out now. I disabled the check globally because I thought it's all over the place. In fact it is just one place.